packages/compass-aggregations/src/components/pipeline-output-options-menu.tsx

```diff
@@ -8,19 +8,29 @@ const pipelineOptionsActions: MenuAction<PipelineOutputOption>[] = [
   { action: 'expand', label: 'Expand all fields' },
 ];
 
+const containerStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  flex: 'none',
+});
+
 export const PipelineOutputOptionsMenu: React.FunctionComponent<{
   onChangeOption: (option: PipelineOutputOption) => void;
   buttonText?: string;
 }> = ({ onChangeOption, buttonText }) => {
   return (
-    <DropdownMenuButton<PipelineOutputOption>
-      data-testid="pipeline-output-options"
-      actions={pipelineOptionsActions}
-      onAction={onChangeOption}
-      buttonText={buttonText ?? 'Output Options'}
-      buttonProps={{
-        size: 'xsmall',
-      }}
-    ></DropdownMenuButton>
+    <div className={containerStyles}>
+      <DropdownMenuButton<PipelineOutputOption>
+        data-testid="pipeline-output-options"
+        actions={pipelineOptionsActions}
+        onAction={onChangeOption}
+        buttonText={buttonText ?? 'Output Options'}
+        buttonProps={{
+          size: 'xsmall',
+          title: buttonText ?? 'Output Options',
+          ['aria-label']: buttonText ?? 'Output Options',
```

Comment:
Instead of duplicating this default value, did you consider moving it to the destructuring of the the `props` above?

```tsx
}> = ({ onChangeOption, buttonText = 'Output Options' }) => {
```

```suggestion
        buttonText={buttonText}
        buttonProps={{
          size: 'xsmall',
          title: buttonText,
          ['aria-label']: buttonText,
```

Reply:
thanks for the prompt! I actually changed this so that the text default is "soft" (can be overridden with an empty string), but the aria label and title will still have a default.